twindle-cli/twitter/transformations/tweets-array-endpoint.js

```diff
@@ -1,73 +1,67 @@
-const {
-    createCustomTweet,
-  } = require("./helpers");
-  
-  const { renderRichTweets, fixUserDescription } = require("./rich-rendering");
-  
-  // const { doTweetsSearch } = require("./tweets-search-request");
-  const { format } = require("../utils/date");
-  
-  /**
-   *
-   * @param {any} responseJSON
-   */
-  function processTweetsArray(responseJSON) {
-    responseJSON = responseJSON.data;
-    const tweets = (responseJSON.data || []).map((resData) => ({
-        ...resData,
-        includes: responseJSON.includes,
-      }));
-    const firstTweet = tweets.filter((tweet) => tweet.id === tweet.conversation_id)[0];
-    const userObject = responseJSON.includes.users.filter(
-        (user) => user.id === firstTweet.author_id
-      )[0];
-    
-    let tweet = renderRichTweets(firstTweet);
-    let user = userObject;
-    
-    let resp = {
-      data: [],
-      common: {},
-    };
-  
-    resp.common.created_at = format(new Date(tweet.created_at), "MMM d, yyyy");
-    resp.common.user = { ...user };
-  
-    resp = fixUserDescription(resp);
-  
-    resp.common.user.profile_image_url = resp.common.user.profile_image_url.replace(
-      "_normal.",
-      "."
+const { createCustomTweet } = require("./helpers");
```

Comment:
What are the changes made in this file? Should @Mira-Alf take a look?

Reply:
I didn't even touch it. Why is it here, I have no idea. lemme see

Reply:
That's weird. I have all that code right here. I'll push again

Reply:
Added back